network.ts

```diff
@@ -9,12 +9,12 @@ export interface TablelandNetworkConfig {
   filecoin: string | number;
   // tableland testnet testnets
   sepolia: string | number;
-  "optimism-goerli": string | number;
+  "optimism-sepolia": string | number;
   "arbitrum-sepolia": string | number;
   maticmum: string | number;
   "filecoin-calibration": string | number;
-  // tableland staging testnets
-  "optimism-goerli-staging": string | number;
+  // tableland testnet devnets (uncomment when needed for admin)
+  // "optimism-sepolia-staging": string | number;
```

Comment:
leaving this here commented out so that it doesn't show up in public api, but available for admin when needed